dts/arm/nuvoton/npcx/npcx7/npcx7-pinctrl.dtsi

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright (c) 2022 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
+	/* Nodes for peripheral device's default pin-muxing and pad properties */
+	/* Host peripheral interfaces */
+	lpc_espi_default: periph-lpc-espi {
+		group0 {
+			pinmux = <&alt1_no_lpc_espi>;
```

Comment:
note: where are these nodes defined? I can't see them here, and this file doesn't include anything

Reply:
Please refer to the npcx pinmux layout file, [npcx-alts-map.dts](https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/arm/nuvoton/npcx/npcx-alts-map.dtsi#L43), which is included in the [npcx soc DT file](https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/arm/nuvoton/npcx7.dtsi#L8).